src/Teambuilder/client.cpp

```diff
@@ -2037,16 +2037,18 @@ void Client::seeInfo(int id, QString tier)
 {
     if (playerExist(id))
     {
-        int challengeId = freeChallengeId();
-        ChallengeDialog *mychallenge = new ChallengeDialog(player(id), team(), ownId(), challengeId);
-        mychallenge->setChallenging(tier);
+        if (!(mychallenges.size() > 4)) {
```

Comment:
Why in `seeInfo` ? Then if someone double-clicks a player on the list nothing happens and they don't know why.

Also that condition in the test... This is better:

```cpp
if (mychallenges.size() >= 5) {
   //too many challenges, do sthing?
} else {
  ...
}
```